pkg/kubefed2/disable.go

```diff
@@ -148,48 +151,104 @@ func DisableFederation(cmdOut io.Writer, config *rest.Config, typeConfigName ctl
 	if err != nil {
 		return errors.Wrap(err, "Failed to get federation clientset")
 	}
+
+	write := func(data string) {
```

Comment:
[Not necessary to be done in this PR] This could be a common utility function both `enable` and `disable` could use.